HIP/hip-657.md

```diff
@@ -102,24 +104,36 @@ message NftMetadataUpdateTransactionBody {
 message TokenCreateTransactionBody {
 ...
     /**
-     * The key which can change the supply of a token. The key is used to sign Token NFT Metadata Update
-     * operations
+     * The key which can change the metadata of a token definition or individual NFT.
      */
-    Key metadataKey = 11;
+    Key metadata_key = 24;
 }
 
 message TokenUpdateTransactionBody {
 ...
     /**
-     * The key which can change the metadata of an NFT. The key is used to sign Token NFT Metadata Update
-     * operations
+     * The key which can change the metadata of a token definition or individual NFT.
      */
-    Key metadataKey = 11;
+    Key metadata_key = 17;
 }
 ```
 
 
-Requirements: The NFT must exist on the Hedera network and should only be allowed to execute if it is signed by the new METADATAKEY. The new metadata must be a valid byte array limited to 100 bytes. The update must be made atomically and be recorded on the Hedera network.
+Requirements: The NFT must exist on the Hedera network and should only be allowed to execute if it is signed by the new METADATA key. The new metadata must be a valid byte array limited to 100 bytes. The update must be made atomically and be recorded on the Hedera network.
```

Comment:
Should not the current metadata key be signing this rather than the new key?

Reply:
When we change keys, we always require both keys to sign the signature.